src/arena/CreateSprintTemplate.js

```diff
@@ -57,6 +58,7 @@ const onDragEnd = (result, columns, setColumns) => {
 };
 
 function CreateSprintTemplate(props) {
+  const theme = useTheme();
```

Comment:
This will make life easier, thank you @vmcodes as well for taking the time to upgrade us to MUI v5!